src/modules/mtev_amqp.c

```diff
@@ -355,7 +355,11 @@ rabbitmq_manage_connection(void *vconn) {
                            conn->queuename ? amqp_cstring_bytes(conn->queuename) : amqp_empty_bytes, 0, 0, 0, 1,
                            amqp_empty_table);
       die_on_amqp_error(amqp_get_rpc_reply(conn->conn), "declaring queue");
-      queuename = amqp_bytes_malloc_dup(r->queue);
```

Comment:
Ths amqp_bytes_malloc_dup function unconditionally calls malloc.  The only reason we could get a null queuename is if that malloc returned NULL.  Is the idea that the r->queue.len == 0?  Usually `malloc(0)` does not return NULL.

Reply:
I don't understand how amqp_queue_declare could succeed and not have a queue name.